testpilot/frontend/static-src/app/models/me.js

```diff
@@ -46,6 +46,8 @@ export default Model.extend({
         .then(result => {
           this.clientUUID = result.clientUUID;
           this.installed = result.installed;
+        }).catch((err) => {
+          console.error('sync-installed failed', err);
```

Comment:
```
testpilot/frontend/static-src/app/models/me.js
  50:11  warning  Unexpected console statement  no-console

✖ 1 problem (0 errors, 1 warning)
```

Not sure if we want to just inline disable that rule for the `console.error()` (ie: `// eslint-disable-line no-console`), or maybe send the error event through GA or some other monitoring service. Or whatever.